muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/rest/WorkspaceRESTService.java

```diff
@@ -2122,9 +2122,9 @@ public Response deleteWorkspaceStudent(@PathParam("WORKSPACEENTITYID") Long work
   public Response listJournalEntries(
       @PathParam("WORKSPACEID") Long workspaceEntityId,
       @QueryParam("userEntityId") Long userEntityId,
-      @QueryParam("workspaceStudentId") String workspaceStudentId
-  ) {
-    
+      @QueryParam("workspaceStudentId") String workspaceStudentId,
+      @QueryParam("firstResult") @DefaultValue ("0") Integer firstResult, 
+      @QueryParam("maxResults") @DefaultValue ("25") Integer maxResults) {
```

Comment:
Purely cosmetic but this method defaults to 25 whereas the backing bean defaults to 20. Would improve Feng Shui if both sides shared the same value, whichever it is :)